FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1033,6 +1036,355 @@ paths:
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        500:
+          description: Server error
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      security:
+      - OAuth2: [fraihmwork.component.write, fraihmwork.component.admin]
+  
+  /monitor/v1/relationship/definition/{name}:
+    delete:  
+      tags:
+      - Relationships
+      - Monitor
+      summary: Deletes a relationship definition from the deployment configuration
+      description: This operation removes a relationship definition from FRAIHMWORK, making it unusable for future relationship instances. This operation cannot be performed on definitions with active relationship instances - they must be deleted first. Note, requests with the fraihmwork.component.write permission may be limited for authorization reasons. See the descriptions in the securitySchemes.OAuth2.flows.clientCredentials.scopes for more details.  
+      operationId: deleteRelationshipDefinition
+      parameters:
+        - in: path
+          name: name
+          required: true
+          schema:
+            type: string
+          description: The name of the relationship definition to delete
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        400:
+          description: Bad request - The relationship definition cannot be removed, likely because of existing relationships that are still using this definition.
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        401:
+          description: Not authenticated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden - client is not authorized to delete this component
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        404:
+          description: Relationship definition with target name not found
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        406:
+          description: Not acceptable - only responds in application/json format
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        500:
+          description: Server error
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      security:
+      - OAuth2: [fraihmwork.component.admin]
```

Comment:
This might be a misunderstanding of the scopes, but does this not allow someone who created a definition to delete it?

Reply:
Should have write - this was before putting in the broader statement above about not allowing deletes when there are active relationships derived from the relationship